Monitoring in a Box/Ansible/host_vars/mib1.yaml

```diff
@@ -13,11 +14,18 @@ NewVmName: "MiB-Host"
 ## New account username
 NewVmUsername: mib
 
-## Pve node sudo password
-ansible_become_pass: [key in pve node sudo password]
-
 ## New account password must be the same as pve node sudo password
 NewVmPassword: "{{ansible_become_pass}}"
 
 ## VM's IP address
 NewVmIP: 10.0.1.240
+
+### Inventory SSH Variables
+# Pve node sudo password
+ansible_become_pass: [key in pve node sudo password]
+# SSH VM username
+ansible_user: "{{NewVmUsername}}"
+# SSH VM IP address
+ansible_host: "{{NewVmIP}}"
+# SSH jumphost option
+ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -q [Key in Jumphost IP] -p [Key in Jumphost Port Number]"'
```

Comment:
If the Ansible script will only be used in your environment, shouldn't it be in `all.yml`? I am confused because the gateway IP and DNS are in `all.yml` because this is for your environment. `ansible_ssh_common_args` should rightfully also be for your environment.

Reply:
Yeah that should be the case, however I did not want to expose the public IP address and also decided to consolidate all the variables which requires editing into a single file to make it easier for the whole setting up process .

Reply:
I understand your train of thought but it's not the right way. When writing code, assume that someone one day will take charge of your code. The poor fella will be confused.

Since this is a global variable for your environment by your definition, it should end up in `all.yml` like the rest. If every single hostvar will use the same `ansible_ssh_common_args`, then it should be treated as a common variable and put either in a groupvar in `all.yml` or `mib_host.yml`.

If you don't wish to expose the public IP, use Ansible Vault to wrap it up as an encrypted variable.

Reply:
Roger. I will make the necessary changes and also do the same for similar variables.

Reply:
Cool thanks!